common/beerocks/bwl/CMakeLists.txt

```diff
@@ -105,6 +105,7 @@ target_include_directories(${PROJECT_NAME}
         $<TARGET_PROPERTY:bcl,INTERFACE_INCLUDE_DIRECTORIES>
         ${PLATFORM_INCLUDE_DIR}
     PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/common>
         $<INSTALL_INTERFACE:include>
```

Comment:
Do we still need the INSTALL_INTERFACE here?

Reply:
We don't, but I would do such cleanups in later commits - we need many more like that. It definitely doesn't harm (INTERFACE is only used by export install).